samples/ControlCatalog.Web/Pages/Index.razor

```diff
@@ -0,0 +1,5 @@
+﻿@page "/"
+
+@using Avalonia.Blazor
+
+<AvaloniaView />
```

Comment:
We might as well separate `<AvaloniaView Content="..."/>` and `<AvaloniaMainView/>` that hosts the view from `ISingleViewApplicationLifetime`. Not required right now.